web/components/issues/issue-layouts/utils.tsx

```diff
@@ -240,3 +244,22 @@ const getCreatedByColumns = (member: IMemberRootStore) => {
     };
   });
 };
+
+/**
+ * This Method finds the DOM element with elementId, scrolls to it and highlights the issue block
+ * @param elementId
+ * @param shouldScrollIntoView
+ */
+export const highlightIssueOnDrop = (
+  elementId: string | undefined,
+  shouldScrollIntoView = true,
+  shouldHighLightWithLine = false
+) => {
+  setTimeout(async () => {
+    const sourceElementId = elementId ?? "";
+    const sourceElement = document.getElementById(sourceElementId);
+    sourceElement?.classList?.add(shouldHighLightWithLine ? HIGHLIGHT_WITH_LINE : HIGHLIGHT_CLASS);
+    if (shouldScrollIntoView && sourceElement)
+      await scrollIntoView(sourceElement, { behavior: "smooth", block: "center", duration: 1500 });
```

Comment:
Isn't 1500ms 2 long for a scroll?

Reply:
1500 because, if the list is very long, the duration is high so that it scrolls to the position and does not time out in between